pkgs/tools/misc/g933-utils-unstable/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, rustPlatform, udev, pkgconfig }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "g933-utils-unstable";
+  version = "2019-08-04";
```

Comment:
unstable is usually part of the version and not of pname.
```suggestion
  pname = "g933-utils";
  version = "unstable-2019-08-04";
```

Reply:
Pushed again, changed the name-folder-pr :+1: